apps/fluent-tester/package.json

```diff
@@ -37,7 +37,8 @@
     "@fluentui-react-native/themed-stylesheet": "^1.1.0",
     "@fluentui-react-native/win32-theme": ">=0.5.1 <1.0.0",
     "@fluentui/react-native": ">=0.21.6 <1.0.0",
-    "react-native-svg": "^11.0.0"
+    "react-native-svg": "^11.0.0",
```

Comment:
I was under the impression the react-native-svg community module only supported iOS/Android. Has that changed to somehow include windows support?

Reply:
I've added native support for it in office win32 netui.